tunnel/dns/file.go

```diff
@@ -73,9 +82,7 @@ func (h *hostFile) AddHostname(hostname string, ip net.IP) error {
 	return nil
 }
 
-func (h *hostFile) RemoveHostname(s string) error {
-	h.mutex.Lock()
-	defer h.mutex.Unlock()
+func (h *hostFile) RemoveHostname(_ string) error {
```

Comment:
Does this ...need to exist? Are we satisfying an interface?

Reply:
Ah, I see it below. If there aren't other impls that need it it is questionable if the interface needs it.

However, no change needed imo.